mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -925,9 +928,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
         //menu items
         switch (item.getItemId()) {
             case R.id.mainmenu_openhab_preferences:
-                Intent settingsIntent = new Intent(this.getApplicationContext(), OpenHABPreferencesActivity.class);
-                startActivityForResult(settingsIntent, SETTINGS_REQUEST_CODE);
-                Util.overridePendingTransition(this, false);
+                Toast.makeText(
```

Comment:
Replace with 
`showMessageToUser(getString(R.string.info_settings_moved), Constants.MESSAGES.TOAST, Constants.MESSAGES.LOGLEVEL.ALWAYS);`
Snackbar won't work here, because it is only visible in MainAcivity.